packages/fxa-admin-server/tsconfig.json

```diff
@@ -15,7 +15,7 @@
     "forceConsistentCasingInFileNames": true,
     "noEmitHelpers": true,
     "importHelpers": true,
-    "typeRoots": ["./types", "node_modules/@types"]
+    "typeRoots": ["./types", "node_modules/@types", "../../node_modules/@types"]
```

Comment:
I wonder if this could exist in a common tsconfig, given how widely used it is. Maybe a future task.

Reply:
Yes, standardizing tsconfig is the next follow-up task. I just had to do a few small changes to get the existing ones working with yarn.